### PR TITLE
Museum information Screen

### PR DESCRIPTION
**Pull request by sam33rdhakal**
Updates 
- Adds museum information screen 
- Adds version info in the information screen

### CODE REVIEW

File: info/src/main/res/layout/fragment_museum_information.xml [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+                    style="@style/BodySansSerifBlackLeft"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="@dimen/marginDouble"
+                    android:layout_marginTop="@dimen/marginDouble"
+                    android:fontFamily="@font/ideal_sans_book"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@id/museumHours"
+                    tools:text="158 Roebling St" />
```
Comment (Cliabhach) [inline info/src/main/res/layout/fragment_museum_information.xml]: Please replace tools-only text with the AIC museum's own address.

File: info/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -8,4 +8,13 @@
     <string name="location_settings">Location Settings</string>
     <string name="support_by">Support By</string>
     <string name="joinUrl" translatable="false">https://sales.artic.edu/memberships?&amp;utm_medium=mobile-app&amp;utm_source=iphone&amp;utm_campaign=join</string>
+
+    <!-- TODO: Remove or change this placeholder text -->
```
[the remark this answers is not in the document]
Comment (sam33rdhakal) [reply info/src/main/res/values/strings.xml]: Nope. That comment is obsolete.

File: info/src/main/java/edu/artic/info/MuseumInformationFragment.kt [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+
+class MuseumInformationFragment : BaseViewModelFragment<MuseumInformationViewModel>() {
+
+    @Inject
+    lateinit var customTabManager: CustomTabManager
+
+    override val viewModelClass: KClass<MuseumInformationViewModel> = MuseumInformationViewModel::class
+
+    override val title: String = "Museum Information"
```
[the remark this answers is not in the document]
Comment (sam33rdhakal) [reply info/src/main/java/edu/artic/info/MuseumInformationFragment.kt]: I don't think we should use `ArticGeneralInfo.title` here. There is no translated `title` field in translations.

File: info/src/main/res/layout/fragment_information.xml [hunk 4]
```diff
@@ -110,6 +110,8 @@
                 style="@style/CardTitleLargeBlack"
                 android:layout_width="0dp"
```
[the remark this answers is not in the document]
Comment (sam33rdhakal) [reply info/src/main/res/layout/fragment_information.xml]: Only margins are different here. We don't have any margin or padding in our current styles. I  think adding new style just for margin/padding won't be fruitful.

[the remark this answers is not in the document]
Comment (Cliabhach) [reply info/src/main/res/layout/fragment_information.xml]: Tbh, I think that'll need to be revisited - margin and padding should be allowed in our styles. This isn't the place to have that conversation, so I'll drop for now.

File: info/src/main/java/edu/artic/info/MuseumInformationFragment.kt (hunk 3 above)
[the remark this answers is not in the document]
Comment (Cliabhach) [reply info/src/main/java/edu/artic/info/MuseumInformationFragment.kt]: Huh, you're right. We'll revisit this when we do the proper review of translation content.

**Pull request merged**